Makefile

```diff
@@ -90,7 +90,9 @@ misspell-fix:
 
 .PHONY: test
 test:
-	$(MAKE) for-all CMD="go test -race ./..."
+	# Temporarily remove -race flag from tests due to an issue in mapstructure failing with -race flag after v1.3.1 https://github.com/mitchellh/mapstructure/issues/290
```

Comment:
do we already have a vivify story in the backlog to add this back in once the issue is resolved so we don't lose track of this?

Reply:
No, I'll add one.